tests/unit/mixins/service-mixin-test.js

```diff
@@ -119,6 +119,11 @@ test('addOptions, authenticated session with authenticated portal options', func
     authMgr: { portal: 'https://session.host.com' },
   });
 
+  // no clue
+  // stub(subject, 'getPortalRestUrl', function (obj) {
```

Comment:
When you do this:
https://github.com/Esri/ember-arcgis-portal-services/blob/be7539547501f1e63c13ac1c4267fb4f9de83459/tests/unit/services/hosted-service-test.js#L3-L4

You'll get `this.stub()` in tests:

https://github.com/Esri/ember-arcgis-portal-services/blob/be7539547501f1e63c13ac1c4267fb4f9de83459/tests/unit/services/hosted-service-test.js#L20-L23

I think case, I don't think you'd even need to `resolve()`, you'd just return the expected URL.

Reply:
swapping in the new `imports` broke ~10 tests. i'm gonna leave the icing off the 🍰 for now.